src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -37,6 +41,12 @@
 #if(MultiOrgAuth)
 using Microsoft.IdentityModel.Tokens;
 #endif
+#if (GenerateApiOrGraph)
+using Company.WebApplication1.Services;
```

Comment:
nit: consider just using the Company.WebApplication1 namespace. Is there a reason it's in a separate namespace?

Reply:
Not really. thanks for the suggestion, @JunTaoLuo